src/guides/v2.4/install-gde/install-quick-ref.md

```diff
@@ -129,6 +129,6 @@ php magento deploy:mode:set developer
 [command line]: {{page.baseurl }}/install-gde/install/cli/install-cli.html
 [Get your authentication keys]: {{page.baseurl }}/install-gde/prereq/connect-auth.html
 [Magento software archive]: {{page.baseurl }}/install-gde/install/get-software.html
-[Readiness check issues]:{{ page.baseurl }}/install-gde/trouble/readiness/tshoot_rc_main.html
+[Readiness check issues]:{{ page.baseurl }}/install-gde/trouble/troubleshooting.html#readiness
```

Comment:
Remove this link. The new KB index topic is an anti-pattern. In fact, this entire topic should be deleted because it's not linked in the TOC.